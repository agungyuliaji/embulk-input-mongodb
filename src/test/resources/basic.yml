in:
  type: mongodb
  uri: mongodb://mongo_user:dbpass@localhost:27017/mydb
  collection: "my_collection"
  projection: '{ "_id": 0, "name": 1, "rank": 1 }'
  sort: '{ rank: 1 }'
out:
  type: file
  path_prefix: ./tmp/basic
  file_ext: csv
  formatter:
    type: csv
    header_line: true
    charset: UTF-8
    newline: CRLF
