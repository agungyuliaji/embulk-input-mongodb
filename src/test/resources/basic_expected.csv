name,rank
obj1,1
obj2,2
obj3,3
obj4,4
obj5,5
obj6,6
obj7,7
obj8,8
obj9,9
